hyperjumptech/monika
# Fix 564 - Crash when stun server is inaccessible

# Monika Pull Request (PR)  

## What feature/issue does this PR add  
1. Closing issue 564
2. When STUN server is inaccesible, monika crashes 

## How did you implement / how did you fix it  
1. Add ping package
2. Before sending request to STUN server,  ping it first.
3. Bypasses stun checking if connection.alive returns false.

## How to test  
1. Run monika normally, ex: `monika -c monika.yml`
2. Ensure you get ip and location: `monika is running from etc.. etc.`
3. Disable the network and run the same command as in step 2. Monika should not crash.
